src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <section>
+      <form action="/search" method="get">
+        <label class="search-bar big" data-qa="big">
+          <img src="./images/Search.svg" alt="search" >
```

Comment:
Need to add image as a background
<img width="496" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/d0fb2f50-64a0-4cc4-84be-042b23d6bc03">
```suggestion
```